src/web/components/Nav/ExpandedMenu/ExpandedMenuToggle/VeggieBurger.tsx

```diff
@@ -60,51 +50,67 @@ const veggieBurgerIcon = ({
     `;
 
     return css`
+        background-color: currentColor;
+        /*
+            IMPORTANT NOTE:
+            we need to specify the adjacent path to the a (current) tag
+            to apply styles to the nested tabs due to the face we use ~
+            to support NoJS
+        */
+        ${`#${navCheckboxId}:checked ~ label & {
+            background-color: transparent;
+        }`}
+
         top: 50%;
         right: 0;
         margin-top: -1px;
         margin-left: auto;
         margin-right: auto;
         ${lineStyles};
-        background-color: ${showExpandedMenu ? 'transparent' : 'currentColor'};
+
         :before {
             ${lineStyles};
             ${beforeAfterStyles};
-            ${showExpandedMenu
-                ? `top: 0;
-            transform: rotate(-45deg);
-            `
-                : 'top: -6px;'};
+            /*
+                IMPORTANT NOTE:
+                we need to specify the adjacent path to the a (current) tag
+                to apply styles to the nested tabs due to the face we use ~
+                to support NoJS
+            */
+            top: -6px;
+            ${`#${navCheckboxId}:checked ~ label & {
+                top: 0;
+                transform: rotate(-45deg);
+            }`}
         }
         :after {
             ${lineStyles};
             ${beforeAfterStyles};
-            ${showExpandedMenu
-                ? `bottom: 0;
-            transform: rotate(45deg);
-            `
-                : 'bottom: -6px;'};
+            /*
+                IMPORTANT NOTE:
+                we need to specify the adjacent path to the a (current) tag
+                to apply styles to the nested tabs due to the face we use ~
+                to support NoJS
+            */
+            bottom: -6px;
+            ${`#${navCheckboxId}:checked ~ label & {
+                bottom: 0;
+                transform: rotate(45deg);
+            }`}
```

Comment:
These are pretty gnarly, is there a way to write them longer-hand, but with greater clarity on what's going on?